src/System.Net.Primitives/src/System/Net/CookieContainer.cs

```diff
@@ -999,58 +998,53 @@ public int GetCookiesCount()
             int count = 0;
             lock (SyncRoot)
             {
-                foreach (CookieCollection cc in _list.Values)
+                foreach (KeyValuePair<string, CookieCollection> pair in _list)
                 {
+                    CookieCollection cc = pair.Value;
                     count += cc.Count;
                 }
             }
             return count;
         }
 
-        public ICollection Values
```

Comment:
I got rid of it and enumerate the collection using `GetEnumerator` in the few places that were using `Values` to avoid the unnecessary `ValueCollection` allocation.